src/main/java/org/linkeddatafragments/solver/ReorderTransformationLDF.java

```diff
@@ -61,7 +61,6 @@ private void initializeMatcher() {
         matcher.addPattern(new Pattern(numTriples, VAR, VAR, VAR));     // ???
     }
 
-    @Override
```

Comment:
Alright, but the `weight` remains, or not? (If not, the method doesn't have a purpose and should perhaps be removed.)